integration-tests/tests/src/tests/alias.ts

```diff
@@ -1,6 +1,6 @@
 ////////////////////////////////////////////////////////////////////////////
 //
-// Copyright 2019 Realm Inc.
+// Copyright 2022 Realm Inc.
```

Comment:
We shouldn't really update Copyright dates. This is when this file was _first_ copyrighted.

Reply:
Well technically it's a new file.  Our linter is doing this for us automatically.

Reply:
Ahh, then it's fine if it's a new file.